adders/paraglide/config/options.ts

```diff
@@ -0,0 +1,49 @@
+import { defineAdderOptions } from '@svelte-add/core';
+
+export const options = defineAdderOptions({
+	availableLanguageTags: {
+		question: 'Which language tags would you like to support?',
```

Comment:
We have another concept of post-install setup instructions. Maybe this would be a good use of that. Rather than asking a question about what language to setup, when the adder is done running we can instruct the user where to further configure their desired languages.

I don't really love the idea of calling out to another CLI because the user has to answer just as many questions in that case.

Reply:
I don't think that calling an external CLI will result in a nice(r) user experience

> If we allowed each tool to go into details about its own configuration, svelte-add would grow a lot and become overwhelming to the user.

Fine grained tool-specific config should absolutely stay out of `svelte-add`, but allowing tools to ask high-level questions is likely unavoidable. Many adders already feel the need to ask the user for config that could theoretically be left to the "post-installation instructions".

- The Tailwind adder asks about adding it's optional `typography` plugin
- Drizzle asks about which database you want & if it should run inside a docker container
- Bootstrap asks about if you want to include it's optional front-end JS bundle

IMO you should have a working app after running "svelte-add [thing]". 

If an i18n library doesn't know which languages to add it either:
- Can't give you a working app
- Has to add some default language-config which likely doesn't match what the user wants

It then has to instruct the user to find and edit the config. Is this really a better user-experience than just asking?

Reply:
These questions change over time, adding churn to the maintenance of svelte-add. 

Setting up a minimal default configuration and then delegating to a config tool of the added item ensures that this stays decoupled and you are in control of your own ui/dx.

The tool (or tools) can be called in succession at the end of the svelte-add call ala 

```
Successfully added inlang with default config to your app, do you want to customize it now? (Y/n)
```

Reply:
That makes sense, I'm open to a flow like that

I do have an implementation question: where would the customization CLI come from? 

If I understand you correctly it would be installed by the adder. How would the versioning of this CLI work? Does the adder install an exact version or just "latest". 
- If exact version: `svelte-add` ends up with churn anyways since updating the CLI requires updating the adder with a new version
- If latest: The same `svelte-add` version may result in different prompts

Reply:
It's hard to find the exact right balance. Some questions are necessary such as what database are you going to use with Drizzle or else we can't setup an even basic integration. But we also want to keep the experience simple so that you get a basic setup without having to think too hard.

I can see the argument here either way. On the one hand you will need to setup the languages eventually. On the other hand, if you're just beginning a new application you might want it to be only in a single language for quite awhile until you expand into new markets. Setting up i18n from the beginning ensures you put your text into translatable strings so that when the time comes you can easily expand to a second language. But I think it's probably less common that an app starts with multiple languages from the beginning and so that's where just defaulting to English rather than asking for a list of languages might make setup simpler.

I suppose things that might be a bit more optional such as which paraglide languages to setup, which tailwind plugins to add, whether to add a dockerfile for your database could be moved into a sort of post-install set of questions. Things that are really necessary for a bare bones integration such as which database to use with Drizzle could remain in the main flow. I think we could play around with this separately though as it's a larger question affecting all adders, so I wouldn't block this PR on it.

> delegating to a config tool of the added item ensures that this stays decoupled and you are in control of your own ui/dx

That doesn't sound like a benefit to me because it means the UX will be different for every adder. In any case, this isn't how any of the adders work today and calling out to a CLI for each tool seems like a separate discussion.